templates/01-hello-username/src/main.js

```diff
@@ -9,12 +9,14 @@ async function initContract() {
   // is hosted at https://wallet.nearprotocol.com
   window.walletAccount = new nearlib.WalletAccount(window.near);
 
-  // Getting the Account ID. If unauthorized yet, it's just empty string.
+  // Getting the Account ID. If currently unauthorized, it's just empty string.
   window.accountId = window.walletAccount.getAccountId();
 
   // Initializing our contract APIs by contract name and configuration.
   window.contract = await near.loadContract(nearConfig.contractName, {
     // NOTE: This configuration only needed while NEAR is still in development
+    // TODO: what configuration?
```

Comment:
Does the comment on line 17 refer to the object that we're inside, or the entire `nearConfig`? Can we give more details or maybe a link to more details about why it will become unnecessary?

Reply:
it refers to `viewMethods`, `changeMethods`

once we have queryable metadata about contract we won't need these

Reply:
Is there a blog post or other documentation about the upcoming queryable metadata that we could link to in this comment?

Reply:
There is NEP in progress:
https://github.com/nearprotocol/NEPs/pull/3